crates/autopilot/src/domain/settlement/auction.rs

```diff
@@ -13,9 +13,6 @@ pub struct Auction {
     pub orders: HashMap<domain::OrderUid, Vec<domain::fee::Policy>>,
     /// Auction external prices
     pub prices: domain::auction::Prices,
-    /// Deadline for an auction solution to be settled, so that it is eligible
-    /// for rewards.
-    pub deadline: domain::eth::BlockNo,
```

Comment:
Maybe we should keep checking against this value? Seems more appropriate on the auction than on a winning solution?

Reply:
Then I move away further from merging this struct to `domain::Auction` that we talked about somewhere.

Reply:
Removed completely deadline. Will be implemented and thought about when implementing CircuitBreaker.